spec/models/user_spec.rb

```diff
@@ -47,6 +47,10 @@
           status:)
   end
 
+  describe "Associations" do
+    it { is_expected.to have_many(:emoji_reactions).dependent(:destroy) }
```

Comment:
I only scoped to the current bug- but I'd like to expand these unit tests for other associations (separately)